crabs/detection_tracking/evaluate_model.py

```diff
@@ -51,52 +53,76 @@ def __init__(
         self.seed_n = args.seed_n
         self.ious_threshold = args.ious_threshold
         self.score_threshold = args.score_threshold
+        self.mlflow_folder = args.mlflow_folder
         self.load_config_yaml()
 
     def load_config_yaml(self):
         with open(self.config_file, "r") as f:
             self.config = yaml.safe_load(f)
 
+    def set_run_name(self):
+        self.run_name = set_mlflow_run_name()
+
+    def setup_logger(self) -> MLFlowLogger:
+        """
+        Setup MLflow logger for testing.
+
+        Includes logging metadata about the job (CLI arguments and SLURM job IDs).
+        """
+        # Assign run name
+        self.set_run_name()
+
+        # Setup logger (no checkpointing)
+        mlf_logger = setup_mlflow_logger(
+            experiment_name="Sep2023_evaluation",
```

Comment:
Should we have experiment_name this as args?

Reply:
yes! I am changing the CLI in a separate PR to better match the training one